jetty-server/src/main/java/org/eclipse/jetty/server/session/JDBCSessionDataStore.java

```diff
@@ -57,9 +57,9 @@
     public static final String NULL_CONTEXT_PATH = "/";
     
     protected boolean _initialized = false;
-    private DatabaseAdaptor _dbAdaptor;
-    private SessionTableSchema _sessionTableSchema;
-    private boolean _schemaProvided;
+    protected DatabaseAdaptor _dbAdaptor;
+    protected SessionTableSchema _sessionTableSchema;
+    protected boolean _schemaProvided;
```

Comment:
I prefer `protected` getter methods rather than `protected` fields because methods can be deprecated and hide better implementation details.